web/styles/zulip.css

```diff
@@ -1736,7 +1736,6 @@ div.focused-message-list {
        default that would otherwise run titles under the search box. */
     .navbar_title,
     .narrow_description {
-        flex: 1 1 0;
```

Comment:
removed because of the;
![image](https://github.com/zulip/zulip/assets/100200105/9b2067d2-2824-4828-9210-9d6e534b8a48)